api/policy.go

```diff
@@ -136,6 +146,20 @@ func UnmarshalPipelineApprovalPolicy(payload string) (*PipelineApprovalPolicy, e
 	return &pa, nil
 }
 
+// AssigneeGroup is the configuration of the assignee group.
+type AssigneeGroup struct {
+	IssueType IssueType          `json:"issueType"`
+	Value     AssigneeGroupValue `json:"value"`
```

Comment:
Seems like there's only one value "PROJECT_OWNER".

Reply:
Yes. DBA is the default value so we make it implicit.